sktm/patchwork.py

```diff
@@ -70,15 +70,6 @@ def __init__(self, url, mbox_sfx, date=None, patch_id=None):
         # Patchwork patch ID for patch objects
         self.patch_id = patch_id
 
-    def __is_patch(self):
-        """
-        Check if the object is a patch.
-
-        Returns:
-            True if the object is a patch object, False if not.
-        """
-        return bool(self.patch_id)
-
```

Comment:
Please leave this one, I'd like to use it for verifying proper arguments are passed and code is correct, using assertions later.